src/overlays/actors/ovl_Boss_Ganon2/z_boss_ganon2.c

```diff
@@ -1909,11 +1909,11 @@ void func_80902524(BossGanon2* this, PlayState* play) {
                     Actor_PlaySfx(&this->actor, NA_SE_EN_MGANON_DAMAGE);
                     Audio_StopSfxById(NA_SE_EN_MGANON_UNARI);
                     this->actor.colChkInfo.health -= 2;
-                    temp_v0_4 = this->actor.colChkInfo.health;
-                    if (temp_v0_4 < 0x15 && this->unk_334 == 0) {
+                    health = this->actor.colChkInfo.health;
+                    if (health < 21 && this->unk_334 == 0) {
                         func_80900818(this, play);
```

Comment:
you can change both occurrences of this to <= 20 and still have it match. just a cleaner number than 21